plugins/contiv/remote_cni_server.go

```diff
@@ -675,7 +693,9 @@ func (s *remoteCNIserver) configureVswitchHostConnectivity(config *vswitchConfig
 		// finish TAP configuration
 		if s.useTAPInterfaces {
 			config.tapHost = s.interconnectTapHost()
-			err = s.vppTxnFactory().Put().LinuxInterface(config.tapHost).Send().ReceiveReply()
+			if !config.configured {
+				err = s.vppTxnFactory().Put().LinuxInterface(config.tapHost).Send().ReceiveReply()
+			}
 			if err != nil {
 				s.Logger.Error(err)
 				return err
```

Comment:
It might be better to do:
+			if !config.configured {
+				err := s.vppTxnFactory().Put().LinuxInterface(config.tapHost).Send().ReceiveReply()
+				if err != nil {
+ 	 				s.Logger.Error(err)
+	  				return err
+				}
+			}

Reply:
Since we return everytime we encounter an error I didn't want to introdouce too many nested levels. But ok I'll change that